libs/brackets-server/hacks/patch/0001-Fix-module-not-found-issue-after-71.patch

```diff
@@ -0,0 +1,154 @@
+-    require("LiveDevelopment/main");
+     require("utils/NodeConnection");
+     require("utils/NodeDomain");
+     require("utils/ColorUtils");
+@@ -161,7 +160,6 @@ define(function (require, exports, module) {
+             DocumentCommandHandlers : require("document/DocumentCommandHandlers"),
+             DocumentManager         : require("document/DocumentManager"),
+             DocumentModule          : require("document/Document"),
+-            DOMAgent                : require("LiveDevelopment/Agents/DOMAgent"),
+             DragAndDrop             : require("utils/DragAndDrop"),
+             EditorManager           : require("editor/EditorManager"),
+             ExtensionLoader         : require("utils/ExtensionLoader"),
+@@ -175,14 +173,10 @@ define(function (require, exports, module) {
+             FindInFiles             : require("search/FindInFiles"),
+             FindInFilesUI           : require("search/FindInFilesUI"),
+             HTMLInstrumentation     : require("language/HTMLInstrumentation"),
+-            Inspector               : require("LiveDevelopment/Inspector/Inspector"),
+             InstallExtensionDialog  : require("extensibility/InstallExtensionDialog"),
+             JSUtils                 : require("language/JSUtils"),
+             KeyBindingManager       : require("command/KeyBindingManager"),
+             LanguageManager         : require("language/LanguageManager"),
+-            LiveDevelopment         : require("LiveDevelopment/LiveDevelopment"),
+-            LiveDevMultiBrowser     : require("LiveDevelopment/LiveDevMultiBrowser"),
+-            LiveDevServerManager    : require("LiveDevelopment/LiveDevServerManager"),
+             MainViewFactory         : require("view/MainViewFactory"),
+             MainViewManager         : require("view/MainViewManager"),
+             Menus                   : require("command/Menus"),
+@@ -191,7 +185,6 @@ define(function (require, exports, module) {
+             PerfUtils               : require("utils/PerfUtils"),
+             PreferencesManager      : require("preferences/PreferencesManager"),
+             ProjectManager          : require("project/ProjectManager"),
+-            RemoteAgent             : require("LiveDevelopment/Agents/RemoteAgent"),
+             ScrollTrackMarkers      : require("search/ScrollTrackMarkers"),
+             UpdateNotification      : require("utils/UpdateNotification"),
+             WorkingSetView          : require("project/WorkingSetView"),
+diff --git a/src/command/DefaultMenus.js b/src/command/DefaultMenus.js
+index 624cc3f..dca712e 100644
+--- a/src/command/DefaultMenus.js
++++ b/src/command/DefaultMenus.js
+@@ -128,8 +128,6 @@ define(function (require, exports, module) {
+         menu.addMenuItem(Commands.TOGGLE_LINE_NUMBERS);
+         menu.addMenuItem(Commands.TOGGLE_WORD_WRAP);
+         menu.addMenuDivider();
+-        menu.addMenuItem(Commands.FILE_LIVE_HIGHLIGHT);
+-        menu.addMenuDivider();
```

Comment:
Live development looks awesome feature and it looks advanced than preview.
Why we should turn off this ?
I suggest you to revive the package related with it instead of removing related code from brackets.

Reply:
Live development feature is for the client program.
So I don't know that it can work correctly on server side.
I will check if the function is working correctly after clearing the problem.

Reply:
@Kwanghyuk-Kim, 
Colud you merge this PR?
I will consider that we can merge the "LiveDevelopment" feature.